deploy/examples/create-external-cluster-resources.py

```diff
@@ -72,7 +72,7 @@ def __init__(self):
         self.dummy_host_ip_map = {}
 
     def _init_cmd_output_map(self):
-        json_file_name = 'test-data/ceph-status-out'
+        json_file_name = 'tests/ceph-status-out'
```

Comment:
The directory is been also changed, 

so I guess we need to visit the outer directory first something like this, `../../tests/ceph-status-out`